scipy/integrate/tests/test_quadpack.py

```diff
@@ -503,6 +503,13 @@ def f(x, y, z):
             error_tolerance=6e-8
         )
 
+    def test_complex(self):
+        def tfunc(x):
+            return np.exp(1j*x)
+
+        assert_quad(quad(tfunc, 0, np.pi/2, complex_func=True),
+                    1+1j, error_tolerance=1e-6)
```

Comment:
There needs to be a test with `full_output=True`.

Reply:
A added a test with `full_output=True` below these lines.